src/scripts.js

```diff
@@ -1,19 +1,76 @@
 
-import {fetchAllData} from './APICalls';
+import {fetchAllData, fetchSingleUser, postData} from './APICalls';
 import Hotel from './classes/hotel';
+import domUpdates from './domUpdates'
 import './css/base.scss';
+import Customer from './classes/customer';
 
+// Global variables
 let hotel;
+let currentUser;
+let sum;
+let rooms;
 
-document.querySelector('button').addEventListener('click', fetchTheData())
+//QuerySelectors
+const searchForm = document.querySelector('form');
 
-function fetchTheData() {
-  let roomsPromise = fetchAllData('rooms');
-  let customersPromise = fetchAllData('customers');
-  let bookingsPromise = fetchAllData('bookings');
-  
-  Promise.all([roomsPromise, customersPromise, bookingsPromise])
-    .then(data => hotel = new Hotel(data[0], data[1], data[2]))
-    .then(() => console.log(hotel))
+
+// Functions
+const loadPage = () => {
+  fetchTheData()
+  .then(data => {
+    hotel = new Hotel(data[0], data[1], data[2])
+    addInfo()
+    defineUser(50)
+    singleCustomerInfo()
+  })
+}
+
+const addInfo = () => {
+  hotel.addRooms();
+  hotel.addBookings();
+  hotel.addCustomers();
+}
+
+const singleCustomerInfo = () => {
+  defineUser(50);
+  hotel.filterBookingsByCustomerID(currentUser);
+  domUpdates.greetUser(currentUser);
+  sum = currentUser.sumUserBookings(hotel)
+  domUpdates.displayUsersExpenses(sum)
+  domUpdates.displayUserBookings(currentUser);
+}
+
+const fetchTheData = () => {
+  const response = Promise.all([fetchAllData('rooms'), fetchAllData('customers'), fetchAllData('bookings')])
+  return response
+}
+
+const defineUser = (userID) => {
+  currentUser = hotel.allCustomers[userID - 1]
+  console.log('user', currentUser)
+  return currentUser
 }
-console.log('This is the JavaScript entry file - your code begins here.');
+
+//eventlisteners
+window.addEventListener('load', () => {
+  loadPage()
+  domUpdates.greetUser(currentUser)
+})
+
+searchForm.addEventListener('submit', (e) => {
+  e.preventDefault();
+  const formData = new FormData(e.target);
```

Comment:
Ooooh, you have this FormData class. I'm github ignorant...is there a way I can snoop into it? We shall talk.